python-libraries/nanover-core/src/nanover/trajectory/frame_data.py

```diff
@@ -289,6 +292,20 @@ class FrameData(metaclass=_FrameDataMeta):
         to_python=_as_is,
         to_raw=_as_is,
     )
+    sparse_user_forces: Array2Dfloat = _Shortcut(  # type: ignore[assignment]
+        key=SPARSE_USER_FORCES,
+        record_type="arrays",
+        field_type="float",
+        to_python=_n_by_3,
+        to_raw=_flatten_array,
+    )
+    sparse_user_index: float = _Shortcut(  # type: ignore[assignment]
```

Comment:
This is an array of float values, not a single float:
```suggestion
    sparse_user_index: List[float] = _Shortcut(  # type: ignore[assignment]
```

Reply:
Good spot!